<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinsettingDialog</class>
 <widget class="QDialog" name="PinsettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>管脚名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="var_name">
       <property name="text">
        <string>newpin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>管脚类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="var_type">
       <item>
        <property name="text">
         <string>bool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>double</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QString</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XYA</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>读写控制：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rwtype">
       <item>
        <property name="text">
         <string>只读</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>只写</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>读/写</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PLC变量名</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="x_label">
        <property name="text">
         <string>X：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="x_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="y_label">
        <property name="text">
         <string>Y：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="y_name"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="a_label">
        <property name="text">
         <string>A：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="a_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
